### PR TITLE
Fix for issue #416 (CDist completely broken - `__explorer=/var/lib/cdist/conf/explorer: Command not found.`)

### PR DESCRIPTION
**Pull request by darko-poljak**
Take a look at code and code comments.
What do you think?

### CODE REVIEW

File: cdist/test/exec/remote.py [hunk 1]
```diff
@@ -39,10 +39,11 @@ def setUp(self):
         user = getpass.getuser()
         remote_exec = "ssh -o User=%s -q" % user
         remote_copy = "scp -o User=%s -q" % user
-        self.remote = remote.Remote(self.target_host, self.base_path, remote_exec, remote_copy)
+        self.remote = remote.Remote(self.target_host, base_path=self.base_path, remote_exec=remote_exec, remote_copy=remote_copy)
 
     def tearDown(self):
-        shutil.rmtree(self.temp_dir)
+        #shutil.rmtree(self.temp_dir)
```
Comment (telmich) [inline cdist/test/exec/remote.py]: Why has this one been commented out?

**Comment by telmich** [comment]
Besides the one comment, it looks good to me!

**Comment by darko-poljak** [comment]
It was a relict from the debugging I missed.
I fixed it.

**Comment by telmich** [comment]
Looks good to me know - @asteven: 4 eye principle: can you give your ok to it, too?

**Comment by asteven** [comment]
Looks good. May be good to rebase from master before merging. The commit history seems to be a mess.

**Comment by darko-poljak** [comment]
@asteven  Yes, I agree, it is a mess. I'll rebase my pull request.

**Comment by darko-poljak** [comment]
@asteven @telmich I made a rebase.

**Comment by asteven** [comment]
Thanks. Looks good.

**Pull request merged**